x/storage/keeper/keeper.go

```diff
@@ -1097,7 +1100,9 @@ func (k Keeper) RejectSealObject(ctx sdk.Context, operator sdk.AccAddress, bucke
 		return errors.Wrapf(types.ErrNoSuchStorageProvider, "SP seal address: %s", operator.String())
 	}
 	if sp.Status != sptypes.STATUS_IN_SERVICE {
-		return sptypes.ErrStorageProviderNotInService
+		if !k.fromMaintenanceSPTestAcct(sp, operator) {
```

Comment:
No need for this, because only seal address is allowed to do this.

Reply:
fixed